tc/app/vault/file/upload

```diff
@@ -26,7 +26,7 @@ done
 
 function _vault_upload() {
   _check_bin_file vault
-  VAULT_TOKEN=$(tcctl vault token)
+  VAULT_TOKEN=$(_vault_get_token)
```

Comment:
you can add here token or username with a password.

If you pass token - you can do everything with token 
If you pass user/password - then with them. 

And now you don't need use a file.

Reply:
I do not understand why? all this is done by the function